cppa_slack_transcript_tracker/utils/slack_listener.py

```
@@ -50,7 +52,16 @@ class SlackListener:
     """Slack Event Listener using Slack Bolt."""
 
     def __init__(self, bot_token=None, app_token=None):
-        self.bot_token = bot_token or get_slack_bot_token()
+        if bot_token:
+            self.bot_token = bot_token
+        else:
+            try:
+                team_id = (
+                    getattr(settings, "SLACK_TEAM_ID", None) or ""
+                ).strip() or None
+            except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)